src/index.js

```diff
@@ -0,0 +1,47 @@
+import './style.css';
+
+const listContainer = document.querySelector('.task-content');
+
+const tasks = [
+  {
+    index: 0,
+    description: 'Morning chores',
+    completed: true,
+  },
+  {
+    index: 1,
+    description: 'After chores',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'Evening chores',
+    completed: false,
+  },
+];
+
+function render() {
+  listContainer.innerHTML = '';
+
+  tasks
```

Comment:
- [ ] I would recommend using proper ES6 syntax instead of the `function` keyword. As it is very useful to write clean code and also it achieve the goal in less code.

```

const render = () => {
 ...
}

Reply:
Thanks for your review, this has been implemented